### PR TITLE
feat(FEV-1090): I as a user want to see the welcome page before video starts

### PR DESCRIPTION
**Pull request by RoyBregman**

### CODE REVIEW

File: src/components/welcome-screen/welcome-screen.scss [hunk 1]
```diff
@@ -1,11 +1,62 @@
...
+      padding-top: 16px;
+    }
+    .download {
+      .icon {
+        height: 24px;
+        width: 24px;
+        margin-right: 4px;
+      }
+      &:hover {
+        background-color: #888888;
```
Comment (semarche-kaltura) [inline src/components/welcome-screen/welcome-screen.scss:42]: background-color: map-get($theme-colors, 'disabled');

File: src/components/welcome-screen/welcome-screen.scss [hunk 2]
```diff
@@ -1,11 +1,62 @@
...
+      }
+      &:focus {
+        background-color: #444444;
```
Comment (semarche-kaltura) [inline src/components/welcome-screen/welcome-screen.scss:45]: move `'focus': #444444` from $secondary-button-colors into $theme-colors and use it in both places

File: src/components/welcome-screen/welcome-screen.tsx [hunk 3]
```diff
@@ -1,21 +1,46 @@
...
 
 interface WelcomeScreenProps {
   allowDownload?: KalturaNullableBoolean;
   welcomeMessage?: string;
+  onDownload: () => void;
   onClose: () => void;
 }
 
-export const WelcomeScreen = ({allowDownload, welcomeMessage, onClose}: WelcomeScreenProps) => {
+export const WelcomeScreen = withText(translates)(({allowDownload, welcomeMessage, onDownload}: WelcomeScreenProps) => {
```
Comment (semarche-kaltura) [inline src/components/welcome-screen/welcome-screen.tsx:21]: here you use translates from your const. Instead you should use translates from component props:
`export const WelcomeScreen = withText(translates)(({allowDownload, welcomeMessage, onDownload, ...translates}: WelcomeScreenProps) => {`

File: src/ivq.tsx [hunk 4]
```diff
@@ -121,7 +122,14 @@ export class Ivq extends KalturaPlayer.core.BasePlugin {
     kalturaCuePointService?.registerTypes([kalturaCuePointService.CuepointType.QUIZ]);
   }
 
-  private _showWelcomeScreen = (quizData: KalturaQuiz) => {
+  private _showWelcomeScreen = () => {
+    const handleDownload = () => {
+      this._player.provider.doRequest([{loader: QuizDownloadLoader, params: {entryId: this._player.sources.id}}]).then((data: Map<string, any>) => {
+        if (data && data.has(QuizDownloadLoader.id)) {
```
Comment (semarche-kaltura) [inline src/ivq.tsx:128]: should we Catch errors here?

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]

File: src/components/welcome-screen/welcome-screen.scss (hunk 1 above)
Comment (RoyBregman) [reply src/components/welcome-screen/welcome-screen.scss:42]: It's the same color but it is not a disabled color :) Not sure we should use the same theme here

File: src/components/welcome-screen/welcome-screen.scss (hunk 2 above)
Comment (RoyBregman) [reply src/components/welcome-screen/welcome-screen.scss:45]: done

File: src/components/welcome-screen/welcome-screen.tsx (hunk 3 above)
Comment (RoyBregman) [reply src/components/welcome-screen/welcome-screen.tsx:21]: done

File: src/ivq.tsx (hunk 4 above)
Comment (RoyBregman) [reply src/ivq.tsx:147]: done

**Pull request merged**